<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Facial Mocap</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QComboBox" name="heads_comboBox">
        <property name="toolTip">
         <string>Select a head</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="process_mocap_pushButton">
        <property name="toolTip">
         <string>Process mocap</string>
        </property>
        <property name="text">
         <string>Process</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bake_animation_checkBox">
        <property name="toolTip">
         <string>Automatically bake on process</string>
        </property>
        <property name="text">
         <string>Bake Animation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Audio Offset:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="audio_offset_lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="frame_audio_pushButton">
        <property name="toolTip">
         <string>Frames the audio range with 30 frame handle</string>
        </property>
        <property name="text">
         <string>Frame Audio Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bake_visible_range_pushButton">
        <property name="toolTip">
         <string>Bake current visible range</string>
        </property>
        <property name="text">
         <string>Bake Visible Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="smooth_curves_pushButton">
        <property name="toolTip">
         <string>Smooth animtion curves</string>
        </property>
        <property name="text">
         <string>Smooth Curves</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Export</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QPushButton" name="export_file_pushButton">
        <property name="toolTip">
         <string>Export this file as FBX</string>
        </property>
        <property name="text">
         <string>Export This File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="ambient_radioButton">
          <property name="toolTip">
           <string>Ambients directory</string>
          </property>
          <property name="text">
           <string>Ambients</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="clickable_radioButton">
          <property name="toolTip">
           <string>Clickables directory</string>
          </property>
          <property name="text">
           <string>Clickables</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>35</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Batch:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="export_head_comboBox">
          <property name="toolTip">
           <string>Character selection for batch export</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="batch_export_pushButton">
        <property name="toolTip">
         <string>Export selected character from selected directory</string>
        </property>
        <property name="text">
         <string>Export Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
